iphone/Classes/TiAppiOSUserActivityProxy.m

```diff
@@ -425,6 +438,73 @@ - (void)resignCurrent:(id)unused
   }
   [_userActivity resignCurrent];
 }
+
+#if IS_XCODE_10
+
+- (NSNumber *)eligibleForPrediction
```

Comment:
Can you make sure all public API's are also in the interface header? It makes it easier to maintain.